docs/mkdocs/docs/azure.md

```diff
@@ -0,0 +1,58 @@
+# Getting started with Azure Blob Storage
+
+[Azure Blob Storage](https://azure.microsoft.com/en-gb/products/storage/blobs) has an extensive set of configuration and access settings.
+There are detailed guides in the [Azure Blob Storage documentation](https://learn.microsoft.com/en-us/azure/storage/blobs/).
+Best practice can depend on your situation.
+This guide is intended as a quick-start, to help you get going with ArcticDB.
+
+You will need an azure account with permission to create storage-accounts.
+- [Install Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) or use the browser based [Cloud Shell](https://shell.azure.com/).
+- If you installed Azure CLI then you will also need to [login](https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli).
+
+### 1. Select a region.  
+
+A region close to your client will mean greater performance.
+You can list your available regions with.
+```sh
+az account list-locations -o table
+```
+
+### 2. Create a resource-group
+
+This is not required but best practice would be to create a new resource-group to try out arcticdb.
```

Comment:
It might be helpful to mention: all the resources in a resource group (RG) can be deleted together in one operation, so it's helpful to create a new RG for each task/project and delete the RG when it's done instead of reusing a resource/RG for different tasks

Reply:
I'll add a comment. You do have to empty storage accounts before you can delete.